hw2/03-count.js

```diff
@@ -1,3 +1,19 @@
 // Add your code here
 
+const input = document.getElementById('userInput');
+const textField = document.getElementById('text');
+const highlightedText = document.getElementById('highlightedText');
+
+const handleKeyDown = function handleUserKeyDown(event) {
+  if (event.keyCode === 13) {
```

Comment:
You may want to consider using keyUp event, it doesn't require to add  event.keyCode and search for the user input as soon as we write the word down.

Reply:
Thank you! I didn't even think about a keyUp event.